config/initializers/01_redis.rb

```diff
@@ -14,3 +14,8 @@
   ConnectionPool.new(size: IdentityConfig.store.redis_attempts_api_pool_size) do
     Redis.new(url: IdentityConfig.store.redis_attempts_api_url)
   end.freeze
+
+REDIS_SOCURE_USERS_POOL =
```

Comment:
Does this need to be a new pool, or could we use `REDIS_THROTTLE_POOL`?

Reply:
Yeah I'll move it to the throttle pool